app/config/index.js

```diff
@@ -0,0 +1,3 @@
+"use strict";
+
+module.exports.Config = require('./config');
```

Comment:
Will be harder to maintain if we do not expose in this way. What if I require config,js file from 10 files. Everywhere I'll have require(./path/config/config.js);
Now I have decides to repackage my config.js, so that its gonna be config/basic/config.js.
In such scenario Ill have to go through all 10 files to update the path.
On the other hand, with exposing from index.js, I'll change only in index,js ./config to ./basic/config